README.md

```diff
@@ -3,6 +3,7 @@
 [![Crate](https://img.shields.io/crates/v/lzma-rs.svg)](https://crates.io/crates/lzma-rs)
 [![Documentation](https://docs.rs/lzma-rs/badge.svg)](https://docs.rs/lzma-rs)
 [![Build Status](https://travis-ci.org/gendx/lzma-rs.svg?branch=master)](https://travis-ci.org/gendx/lzma-rs)
+![Minimum rust 1.32](https://img.shields.io/badge/rust-1.32%2B-orange.svg)
```

Comment:
Why is there one less pair of square brackets in this line compared to the others? Is there a relevant link to Rust 1.32 to put here (release notes maybe)?

Reply:
https://github.com/rust-lang/rfcs/blob/master/text/2495-min-rust-version.md perhaps?

Otherwise yes, https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1320-2019-01-17.
But I usually just leave this without a link, for the lack of a service which shows the MSRV of a whole tree of crates/dependencies.

Reply:
I'd say the second link (release notes) could be relevant then.